disnake/app_commands.py

```diff
@@ -206,6 +206,15 @@ class Option:
         The minimum value permitted.
     max_value: Union[:class:`int`, :class:`float`]
         The maximum value permitted.
+    min_length: Optional[:class:`str`]
+        The minimum length for this option if the type of this Option is :attr:`OptionType.string`.
```

Comment:
Should note maximum length of 6000 chars in these docstrings as well

Reply:
I intentionally left it out here as that length is not currently documented officially.